README.md

```diff
@@ -21,6 +21,12 @@ $ npm install
 $ npm run dev
 ```
 
+If you wish to use the latest version of the webpack template, do the following instead:
```

Comment:
Please add  a bold warning that this branch is not considered stable and can contain bugs or not build at all, so use at your own risk.